src/locales/de_DE.json

```diff
@@ -72,59 +85,60 @@
 "label.action.create.vm.processing": "VM wird erstellt....",
 "label.action.create.volume": "Volumen erstellen",
 "label.action.create.volume.processing": "Volumen wird erstellt....",
-"label.action.delete.account": "Benutzerkonto l\u00f6schen",
-"label.action.delete.account.processing": "Konto wird gel\u00f6scht....",
-"label.action.delete.cluster": "L\u00f6schen des Clusters",
-"label.action.delete.cluster.processing": "Cluster wird gel\u00f6scht....",
-"label.action.delete.disk.offering": "Festplattenangebot l\u00f6schen",
-"label.action.delete.disk.offering.processing": "Festplattenangebot wird gel\u00f6scht....",
-"label.action.delete.domain": "L\u00f6schen der Domain",
-"label.action.delete.domain.processing": "Dom\u00e4ne wird gel\u00f6scht....",
-"label.action.delete.firewall": "Firewall-Regel l\u00f6schen",
-"label.action.delete.firewall.processing": "Firewall wird gel\u00f6scht....",
-"label.action.delete.ingress.rule": "Zutrittsregel l\u00f6schen",
-"label.action.delete.ingress.rule.processing": "Zutrittsregel wird gel\u00f6scht....",
-"label.action.delete.ip.range": "IP-Bereich l\u00f6schen",
-"label.action.delete.ip.range.processing": "IP-Bereich wird gel\u00f6scht....",
-"label.action.delete.iso": "ISO l\u00f6schen",
-"label.action.delete.iso.processing": "ISO wird gel\u00f6scht....",
-"label.action.delete.load.balancer": "Lastverteilerregel l\u00f6schen",
-"label.action.delete.load.balancer.processing": "Lastverteiler wird gel\u00f6scht....",
-"label.action.delete.network": "L\u00f6schen des Netzwerks",
-"label.action.delete.network.processing": "Netzwerk wird gel\u00f6scht....",
-"label.action.delete.nexusvswitch": "Nexus 1000v l\u00f6schen",
+"label.action.delete.account": "Benutzerkonto löschen",
+"label.action.delete.account.processing": "Konto wird gelöscht....",
+"label.action.delete.backup.offering": "Backup-Angebot löschen",
+"label.action.delete.cluster": "Löschen des Clusters",
+"label.action.delete.cluster.processing": "Cluster wird gelöscht....",
+"label.action.delete.disk.offering": "Festplattenangebot löschen",
+"label.action.delete.disk.offering.processing": "Festplattenangebot wird gelöscht....",
+"label.action.delete.domain": "Löschen der Domain",
+"label.action.delete.domain.processing": "Domäne wird gelöscht....",
+"label.action.delete.firewall": "Firewall-Regel löschen",
+"label.action.delete.firewall.processing": "Firewall wird gelöscht....",
+"label.action.delete.ingress.rule": "Zutrittsregel löschen",
```

Comment:
Ingress-Regel löschen

Reply:
Good question. I know Ingress and Egress are known names but we should find a german comparable name. Maybe "Regel für den eingehenden Datenverkehr löschen" ... ?